main.js

```diff
@@ -844,28 +844,48 @@ require('es6-promise').polyfill();
     }
   }
 
-  proto.requestMessagesSinceTimestamp = function requestMessagesSinceTimestamp(lastTimestamp, quantity, withGroups){
-    // if (this.socketConnection == null) {
-    //
-    // }
+  proto.unsend = function unsend(message){
```

Comment:
make it so the function receives only the messageId, and the senderId you get it from the sessionId